experiments/langchain_lab/chatbot_demo/__init__.py

```diff
@@ -0,0 +1,59 @@
+"""Samples of Chatbot."""
+
+import dotenv
+from langchain.prompts import (
+    PromptTemplate,
+    SystemMessagePromptTemplate,
+    HumanMessagePromptTemplate,
+    ChatPromptTemplate,
+)
+from langchain.schema.runnable import RunnablePassthrough
+from langchain_community.vectorstores import Chroma
+from langchain_community.chat_models import ChatOpenAI
+from langchain_core.output_parsers import StrOutputParser
+
+
+dotenv.load_dotenv()
+
+
+def get_rag_chatbot(
+    vectorstore, most_relevant_doc_count: int = 10):
+  """Get RAG Chatbot."""
+  qa_system_template_str = """Your job is to play senior engineer
```

Comment:
is there a better way to write the triple quote strings to also follow the indention inside the function?

Reply:
Any suggestion here? I add more content into the docstr and please help me to review it again. Thanks!

Reply:
I am also not sure, but found this one useful, FYI and you can decide it you like to adapt to it or not.
```python
import textwrap
textwrap.dedent(qa_system_template_str)
```

Reference: [How to remove extra indentation of Python triple quoted multi-line strings?](https://stackoverflow.com/questions/1412374/how-to-remove-extra-indentation-of-python-triple-quoted-multi-line-strings)

Reply:
Thanks for the sharing. I give `textwrap` a test and it looks good:
```python
>>> print(qa_system_template_str)

  Your job is to play senior engineer
  to answer questions about how to use the repo `Cockatoo.AI` on Github.

  Use the following context to answer questions. Be as detailed as possible,
  but don't make up any information that's not from the context.

  If you don't know an answer, say you don't know.

  {context}
  
>>> print(textwrap.dedent(qa_system_template_str))

Your job is to play senior engineer
to answer questions about how to use the repo `Cockatoo.AI` on Github.

Use the following context to answer questions. Be as detailed as possible,
but don't make up any information that's not from the context.

If you don't know an answer, say you don't know.

{context}

>>> print(textwrap.dedent(qa_system_template_str).strip())
Your job is to play senior engineer
to answer questions about how to use the repo `Cockatoo.AI` on Github.

Use the following context to answer questions. Be as detailed as possible,
but don't make up any information that's not from the context.

If you don't know an answer, say you don't know.

{context}
```
I have made modification correspondingly. Please help me to review again. Thanks!